OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -92,7 +92,26 @@ public function __construct(HttpClientInterface $httpClient, HttpUtils $httpUtil
 
         // Resolve merged options
         $resolver = new OptionsResolver();
+
         $this->configureOptions($resolver);
+
+        // See Symfony\Component\Form\ResolvedFormType logic
+        $baseClasses = array(
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\AbstractResourceOwner',
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\GenericOAuth1ResourceOwner',
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\GenericOAuth2ResourceOwner',
+        );
+
+        $reflector = new \ReflectionMethod($this, 'configureOptions');
+        $isOldOverwritten = !in_array($reflector->getDeclaringClass()->getName(), $baseClasses, true);
+
+        $reflector = new \ReflectionMethod($this, 'setupOptions');
+        $isNewOverwritten = !in_array($reflector->getDeclaringClass()->getName(), $baseClasses, true);
+
+        if ($isOldOverwritten && !$isNewOverwritten) {
+            trigger_error('The AbstractResourceOwner::configureOptions() method is deprecated since version 0.4 and will be removed in 1.0. Use setupOptions() instead.', E_USER_DEPRECATED);
```

Comment:
Well, seems not: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/ResolvedFormType.php#L228

;-)